clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -524,4 +527,42 @@ public static ByteBuffer ensureCapacity(ByteBuffer existingBuffer, int newLength
         return existingBuffer;
     }
 
+    /*
+     * Creates a set
+     * @param elems the elements
+     * @param <T> the type of element
+     * @return Set
+     */
+    public static <T> HashSet<T> mkSet(T... elems) {
```

Comment:
nit: **I** would name this method ``newSet``and make it return ``Set`` instead of a ``HashSet``, but, again, up to you. :) 

ps: btw, once Guava is incorporated into Kafka project, the ``Sets`` helper class has methods to replace this one. ;)